meinberlin/templates/account/password_reset.html

```diff
@@ -1,29 +1,29 @@
 {% extends "account/base.html" %}
 {% load i18n account %}
 
-{% block head_title %}{% translate "Password Reset" %}{% endblock %}
+{% block head_title %}
+    {% translate "Password Reset" %}
+{% endblock head_title %}
 
 {% block content %}
-    <h1>{% translate "Password Reset" %}</h1>
-    {% if user.is_authenticated %}
-        {% include "account/snippets/already_logged_in.html" %}
-    {% endif %}
+    <div class="narrow-wrapper">
+        <h1>{% translate "Password Reset" %}</h1>
 
-    <p>{% translate "Forgotten your password? Enter your e-mail address below, and we'll send you an e-mail allowing you to reset it." %}</p>
+        <p>{% translate "Forgot password? Enter your email address and you will receive an email with which you can reset your password. Please contact us if you have any problems resetting your password." %}</p>
```

Comment:
When are you updating messages, in general? As in translations? Should this happen in this PR?